interface/wx/maskededit.h

```diff
@@ -0,0 +1,935 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        maskededit.h
+// Purpose:     interface of wxMaskedEdit
+// Author:      wxWidgets team
+// RCS-ID:      $Id$
```

Comment:
Shouldn't use this any more -- but should have a copyright statement.

Reply:
@vadz ,
Looking at i.e., interface/wx/button.h, I see it has the same header.
Was it changed relatively recently?